coffea/processor/dataframe.py

```diff
@@ -13,26 +12,21 @@ class LazyDataFrame(MutableMapping):
 
     Parameters
     ----------
-        tree : uproot4.TTree
+        tree : uproot.TTree
             Tree to read
         entrystart : int, optional
             First entry to read, default: 0
         entrystop : int, optional
             Last entry to read, default None (read to end)
         preload_items : iterable
             Force preloading of a set of columns from the tree
-        flatten : bool
```

Comment:
I don't think flatten is used in any tutorials now, but might be worth checking

Reply:
it's not